Formula/znapzend.rb

```diff
@@ -20,6 +21,49 @@ def install
     system "make", "install"
   end
 
+  def post_install
+    (var/"log/#{name}").mkpath
+    (var/"run/#{name}").mkpath
+  end
+
+  plist_options(:startup => true, :manual => "znapzend --daemonize")
```

Comment:
the parentheses are not idiomatic and I assume znapzend needs sudo to start manually

Reply:
> the parentheses are not idiomatic

I don’t really see an alternative; neither could I find any in the core tap:

```
grep -R 'var.*log.*mkpath' "$(brew --repository)/Library/Taps/homebrew/homebrew-core"
```

> assume znapzend needs sudo to start manually

In most cases, yes. Good point.

Reply:
you mean like
```
stubby.rb:  plist_options :startup => true, :manual => "sudo stubby -C #{HOMEBREW_PREFIX}/etc/stubby/stubby.yml"
```